app/src/main/java/com/example/finportfolio/fragments/assetdetails/AssetDetailsFragment.kt

```diff
@@ -60,4 +68,29 @@ class AssetDetailsFragment : BaseFragment<FragmentAssetDetailsBinding>() {
             }
         }
     }
+
+    private fun makeAlertDialog(): AlertDialog.Builder? {
+        val builder = this.context?.let { context -> AlertDialog.Builder(context) }
```

Comment:
```suggestion
        val builder = requireContext()?.let { context -> AlertDialog.Builder(context) } ?: return null
```

избавимся от нулабельности далее и не будем выполнять лишние действия